src/views/includes/interior-navigation/base/link-with-children.blade.php

```diff
@@ -0,0 +1,57 @@
+<div
+    @if(data_get($link, 'expandable', false))
+        x-data="{
+            expanded: {{ data_get($link, 'expanded', false) ? 'true' : 'false' }}
+        }"
+    @endif
+>
+    <button
+        class="
+            w-full flex items-center justify-between text-left
+            @if(data_get($link, 'active', false))
+                px-2 border-l-2 transition-colors text-gray-900 font-bold border-red-600
+            @else
+                px-2 border-l-2 transition-colors border-transparent hover:text-gray-900
+            @endif
+        "
+        @if(data_get($link, 'expandable', false))
+            x-on:click="expanded = ! expanded"
+        @endif
+    >
```

Comment:
If the item is not "expandable", let's change this element to something generic instead of a button.